lib/smart_udap_harmonization_test_kit/smart_udap_token_response_scope_test.rb

```diff
@@ -17,36 +17,39 @@ class SMART_UDAP_TokenResponseScopeTest < Inferno::Test # rubocop:disable Naming
           :udap_token_endpoint,
           :udap_client_id
 
-    output :smart_credentials,
-           :id_token,
+    output :id_token,
            :refresh_token,
            :access_token,
            :expires_in,
            :patient_id,
            :encounter_id,
-           :received_scopes
+           :received_scopes,
+           :smart_auth_info
 
     run do
       assert_valid_json(udap_auth_code_flow_token_exchange_response_body)
       token_response_body_parsed = JSON.parse(udap_auth_code_flow_token_exchange_response_body)
 
-      output smart_credentials: {
-        refresh_token: token_response_body_parsed['refresh_token'],
-        access_token: token_response_body_parsed['access_token'],
-        expires_in: token_response_body_parsed['expires_in'],
-        client_id: udap_client_id,
-        client_secret: '',
-        udap_auth_code_flow_token_retrieval_time:,
-        token_url: udap_token_endpoint
-      }.to_json
+      smart_auth_info = Inferno::DSL::AuthInfo.new(
+        {
+          refresh_token: token_response_body_parsed['refresh_token'],
+          access_token: token_response_body_parsed['access_token'],
+          expires_in: token_response_body_parsed['expires_in'],
+          client_id: udap_client_id,
+          issue_time: udap_auth_code_flow_token_retrieval_time,
+          requested_scopes: udap_authorization_code_request_scopes,
```

Comment:
one issue now, the `requested_scopes` for openid  should be `:udap_auth_code_flow_registration_scope` instead of  `udap_authorization_code_request_scopes` looking at the original implementation.

Reply:
Where do you see that?

Reply:
[deleted line 25 on smart_udap_openid_connect_group.rb](https://github.com/inferno-framework/smart-udap-harmonization-test-kit/pull/14/files#diff-6be9a0fbebfd87f053c2d093d8cba4cbaa381876138efd656cf996c6641a47f3L25)

Reply:
Ok, this has been updated.